### DISCUSSION TITLE
airflow not running

### DISCUSSION THREAD

**Discussion by AliVelayati1375** [apache/airflow#31492] [General]
I started Airflow with Linux subsystem step by step, but it gave me the following error message on the browser. Thank you for your help
Something bad has happened.

Airflow is used by many users, and it is very likely that others had similar problems and you can easily find
a solution to your problem.

Consider following these steps:

  * gather the relevant information (detailed logs with errors, reproduction steps, details of your deployment)

  * find similar issues using:
     * GitHub Discussions
     * GitHub Issues
     * Stack Overflow
     * the usual search engine you use on a daily basis

  * if you run Airflow on a Managed Service, consider opening an issue using the service support channels

  * if you tried and have difficulty with diagnosing and fixing the problem yourself, consider creating a bug report.
    Make sure however, to include all relevant details and results of your investigation so far.

Python version: redact
Airflow version: redact
Node: redact
-------------------------------------------------------------------------------
Error! Please contact server admin.

**Comment by potiuk** [comment]
> gather the relevant information (detailed logs with errors, reproduction steps, details of your deployment)

**Comment by potiuk** [comment]
Look at the logs.
